scripts/distro/kubeadm/distro.sh

```diff
@@ -196,4 +196,28 @@ EOF
 function kubeadm_api_is_healthy() {
     curl --globoff --noproxy "*" --fail --silent --insecure "https://$(kubernetes_api_address)/healthz" >/dev/null
 }
+
+function kubeadm_conf_api_version() {
+    
+    # Get version from runtime
+    # Enforce the use of kubeadm.k8s.io/v1beta api version beginning with Kubernetes 1.26+
+    local kubeadm_v1beta3_min_version=
+    kubeadm_v1beta3_min_version="26"
+    if [ -n "$KUBERNETES_TARGET_VERSION_MINOR" ]; then
```

Comment:
`KUBERNETES_TARGET_VERSION_MINOR` is always set so there is no need to check if it is _set_.

Reply:
@emosbaugh It turns out I need this check. It's a predicate to determine whether I need to grab the version from the installer runtime or from an existing cluster (e.g. the [weave-to-flannel migration task](https://github.com/replicatedhq/kURL/pull/4407/files#diff-ee341a571b062fdbc0e72cd063cfaef84629998edace18b631db9efcf2963809R797)).